config/clients/java/template/.github/workflows/main.yml.mustache

```diff
@@ -28,8 +28,11 @@ jobs:
 
   publish:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     if: startsWith(github.ref, 'refs/tags/v')
-    needs: [test]
+    needs: [build]
```

Comment:
We need to have test in the workflow too. Release should not run before the tests have run successfully

Reply:
No wait, I take that back... It would only run tests and lints but only the sdk-generator project is running integration tests so far

Reply:
https://github.com/openfga/sdk-generator/pull/203/commits/131f6e8f20488bc323a530de2ca383ef05281716

Reply:
Let me know if you'd want me to add integ tests too... That would need some minor translation from Makefile to GitHub CI

Reply:
We can leave it out for now